db/migrate/20161031204903_add_tracer_support.rb

```diff
@@ -0,0 +1,13 @@
+class AddTracerSupport < ActiveRecord::Migration
+  def change
+    create_table :katello_host_tracers do |t|
+      t.references 'host', :null => false, :index => true
+      t.string 'application'
+      t.string 'helper'
+      t.string 'app_type'
```

Comment:
would add :null => false to the appropriate entries here to match the validations